src/storage/table_codec.rs

```diff
@@ -98,6 +98,45 @@ impl TableCodec {
                     })
             })
     }
+
+
+    /// Key: RootCatalog_0_TableName
+    /// Value: TableName
+    pub fn encode_root_table(table_name: &str) -> Option<(Bytes, Bytes)> {
+        let key = format!(
+            "RootCatalog_{}_{}",
+            BOUND_MIN_TAG,
+            table_name,
+        );
+
+        bincode::serialize(&table_name).ok()
```

Comment:
oops, On the Index branch I am currently developing, I added this to the TypeError for bincode errors.
``` rust
#[error("bindcode")]
    Bincode(
        #[source]
        #[from]
        Box<bincode::ErrorKind>
    )
```
So I can throw the error directly instead of returning Option:
![image](https://github.com/KipData/KipSQL/assets/91525956/8e3bc3be-67a5-410a-9f05-ed676ff22c0e)

Reply:
When will you fix it to prevent conflict?